### PR TITLE
Feat: added login and signup page

### PR DESCRIPTION
**Pull request by sakleprajwal**
Login and Signup page is added to the E-commerce website.
Preview:

https://user-images.githubusercontent.com/40752468/154791475-b4d48967-a60e-47be-933e-7b0586cf6941.mp4

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </div>
+            <div class="checkbox-link-input flex-row">
+                <div class="checkbox-input">
+                    <input type="checkbox" id="term&condition">
+                    <label for="term&condition">Remember me</label>
+                </div>
+                <a href="">Forgot your password?</a>
+            </div>
+            <button class="btn text-button">Login</button>
+            <button class="btn login-link-button"><a href="">Create New Account</a></button>
```
Comment (Gautam2305) [inline login.html]: Don't leave href attribute blank like this...... it should be given "#" if not assigned to a link

**Review by Gautam2305 — COMMENTED** [review]
Rest everything looks good.....merge after another review

File: login.html (hunk 1 above)
Comment (sakleprajwal) [reply login.html]: It is supposed to have the signup link. Thanks for highlighting.

Comment (Gautam2305) [reply login.html]: Oh....then its fine👍🏻

**Review by Mahi24-prog — APPROVED** [review]
Looks good

**Pull request merged**